.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - alpha
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: current
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+          run_install: true
+      - name: Install dependencies
```

Comment:
This is not the "Install dependencies" step

Reply:
Good point! We fixed it in [d3091ee](https://github.com/warp-ds/core/pull/3/commits/d3091ee6951ef423d4c8504f5b59a2c65dc6f9c9)